pkg/taskrunmetrics/metrics_test.go

```diff
@@ -615,42 +615,38 @@ func TestTaskRunIsOfPipelinerun(t *testing.T) {
 func TestRecordCloudEvents(t *testing.T) {
 	for _, c := range []struct {
 		name          string
-		taskRun       *v1beta1.TaskRun
+		taskRun       *v1.TaskRun
 		expectedTags  map[string]string
 		expectedCount float64
 	}{{
 		name: "for succeeded task",
-		taskRun: &v1beta1.TaskRun{
+		taskRun: &v1.TaskRun{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      "taskrun-1",
 				Namespace: "ns",
 				Labels: map[string]string{
 					pipeline.PipelineLabelKey:    "pipeline-1",
 					pipeline.PipelineRunLabelKey: "pipelinerun-1",
 				},
+				Annotations: map[string]string{
```

Comment:
this looks odd -- isn't this annotation only added when converting a v1beta1 TaskRun that has the now removed status.cloudEvents field? Do we really need to add it to the test for it to pass?